10-Summation_of_primes.js

```diff
@@ -0,0 +1,29 @@
+// The sum of the primes below 10 is 2 + 3 + 5 + 7 = 17.
+//
+// Find the sum of all the primes below two million.
+
+function primesSum() {
+  const maxPrimeValue = 2e+6;
+  var sum = 0;
+
+  for(let i = 2; i <= maxPrimeValue; i++) {
```

Comment:
same, https://stackoverflow.com/questions/4445706/post-increment-and-pre-increment-concept